examples/playground/src/actions/HelperActions.js

```diff
@@ -0,0 +1,20 @@
+import { analytics, trackerId } from '../config/base-config';
+
+const ReactGA = require('react-ga');
+
+var HelperActions = {
```

Comment:
This should probably be something like `GoogleAnalyticActions`  or `GAActions` or w/e. 

Ideally, it would be a class that it exported with a `logPageView` method. 

This way we would could ...
```javascript
import { GA } from 'HelperActions;

// Another File

import { OtherHelperAction } from 'HelperActions';
```

You know what I mean?